QUARKUS-1415.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-1415 Reactive REST Client improvements
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1415
+
+Test plan covers RESTEasy Reactive functionality provided by `quarkus-resteasy-reactive` and `quarkus-rest-client-reactive` extensions,
```

Comment:
[QUARKUS-1415](https://issues.redhat.com/browse/QUARKUS-1415) is about Reactive REST Client improvements